step-controller-plugins/step-controller-plugins-measurement/src/main/java/step/plugins/measurements/MeasurementPlugin.java

```diff
@@ -277,7 +325,20 @@ private boolean isArtefactInstrumented(AbstractArtefact artefactInstance) {
 	private void enrichWithNodeAttributes(Measurement measurement, ReportNode node) {
 		measurement.setExecId(node.getExecutionID());
 		measurement.addCustomField(RN_ID, node.getId().toString());
-		measurement.setStatus(node.getStatus().toString());
+		ReportNodeStatus nodeStatus = node.getStatus();
+		if(nodeStatus == ReportNodeStatus.RUNNING) {
+			// For live measures, the node status is still RUNNING, so we rely on the measure status
+			if (measurement.getStatus() == null) {
+				// this should never be the case, as all measures should now have a status (PASSED by default), but better safe than sorry
+				measurement.setStatus(nodeStatus.name());
+			}
+		} else {
+			// "old-style" measures (attached to output) take the node status, EXCEPT if the measure itself indicates a non-default status, i.e. a failure
+			String mStatus = measurement.getStatus();
+			if (mStatus == null || mStatus.equals(Measure.Status.PASSED.name())) {
```

Comment:
As discussed: 
- If mStatus is null => take the nodeStatus
- If mStatus is set => take it as defined

The step-api should be changed accordingly:
- Legacy KW api => null values for the status accepted and set by default (Measurement constructor and stopMeasure), a status can be optionally set. 
- Live reporting API => Set to PASSED by default, null values should never happen